### DISCUSSION TITLE
How can i change the input pins for a inmp441 i2s input?

### DISCUSSION THREAD

**Discussion by f0m3** [pschatzmann/arduino-audio-tools#793] [Q&A] [answered]
Im trying to understand the examples.
Unfortunatly it isn't described how to change the standard pins of the i2s input to the esp32.

**Comment by f0m3** [comment, marked as answer]
i just found out but maybe that helps someone else: 
in void setup() add 

```
  // i2s pins used on ESP32
  config.pin_bck = 27;
  config.pin_ws = 25;
  config.pin_data = 26;
```

before 
` i2sStream.begin(config);`

**Reply by pschatzmann** [reply]
This is described in the ADC chapter: https://github.com/pschatzmann/arduino-audio-tools/wiki/External-ADC
Sometimes reading the documentation helps!

I also suggest to have a look at the class documentation: there are quite some more parameters available
